tests/actions/internal/test_discovery_extension_request_handler.py

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+#
+
+import unittest
+
+
+class TestDiscoveryExtensionsRequestHandler(unittest.TestCase):
+    pass
```

Comment:
I'm curious why the new test classes here that don't seem to do anything?

Reply:
Placeholders, we should fill them out.